tools/testnet/files/dockerfiles/mkkeystore/mkkey.py

```diff
@@ -15,7 +15,7 @@
     from ethereum.tools import keys
 except ImportError:
     from ethereum.keys import make_keystore_json, sha3, encode_hex
-    from ethereum import keys
+    from ethereum.tools import keys
```

Comment:
This was a fallback for `ethereum<2.0`, but I guess the `ImportError` block can now be removed completely.